python/pyspark/cloudpickle/__init__.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import
+
+
+from pyspark.cloudpickle.cloudpickle import *  # noqa
```

Comment:
Everything is same as the original cloudpickle except these two line diff. We should add `pyspark` to import.